QUEST_JOBSTEP.tsv

```diff
@@ -30,18 +30,18 @@ QUEST_JOBSTEP_20150317_000029	So you want to learn the way of the arrow from me.
 QUEST_JOBSTEP_20150317_000030	Evoniphon...I will never forgive him.
 QUEST_JOBSTEP_20150317_000031	Hunter Master
 QUEST_JOBSTEP_20150317_000032	You collected all the books. Good job. {nl}I will teach you the way of a Hunter as promised. 
-QUEST_JOBSTEP_20150317_000033	You got some sure results. I will teach you from now on. {nl}I hope you live up to my expectations. 
+QUEST_JOBSTEP_20150317_000033	You got some good results. I will teach you from now on. {nl}I hope you live up to my expectations. 
 QUEST_JOBSTEP_20150317_000034	I heard the bats in Crystal Mines 1st Floor have special venom. {nl}I'm sure it's the venom used by the assassin Evoniphon. Bring me that venom.
 QUEST_JOBSTEP_20150317_000035	Barbarian Master
-QUEST_JOBSTEP_20150317_000036	You can understand an opponent's abilities by facing them sword on sword. {nl}Prove to me that you are a true warrior, and I will lead you in becoming a real Barbarian. 
+QUEST_JOBSTEP_20150317_000036	You can understand an opponent's abilities by facing them sword to sword. {nl}Prove to me that you are a true warrior, and I will lead you in becoming a real Barbarian. 
 QUEST_JOBSTEP_20150317_000037	You indeed qualify as a warrior. {nl}I will make you become a true Barbarian. 
 QUEST_JOBSTEP_20150317_000038	I understand that you want to be stronger with a spear. {nl}Complete the mission I assign to you, and I will teach you my spearmanship.
 QUEST_JOBSTEP_20150317_000039	
 QUEST_JOBSTEP_20150317_000040	I understand your will to become stronger. {nl}I think you have what it takes to be good with spears.
 QUEST_JOBSTEP_20150317_000041	Psychokino Master
-QUEST_JOBSTEP_20150317_000042	If you want to understand Psychokino, how about you work as my assistant.{nl}I will guarantee you that it is an attracting job.
-QUEST_JOBSTEP_20150317_000043	Please first defeat Matsums from Spine Heart Dale and collect flower alcohol.{nl}They break easily so it's hard to get the decent ones.
-QUEST_JOBSTEP_20150317_000044	Good. Flower alcohol of Matsum is okay.{nl}You will see something interesting soon.
+QUEST_JOBSTEP_20150317_000042	If you want to understand Psychokino, then how about you work as my assistant?{nl}I will guarantee you that it is an attractive job.
+QUEST_JOBSTEP_20150317_000043	Please first defeat Matsums from Spine Heart Dale and collect their flower alcohol.{nl}They break easily so it's hard to get the decent ones.
```

Comment:
Please is sorta pointless but none the less its coming from an NPC so it should it is fine for now. If it was coming from the game itself than please should be removed. I think "Please, first Defeat Matsums..." is fine for now although it does sound kinda odd, which is why I commented on it. Refer to this issue: https://github.com/Treeofsavior/EnglishTranslation/issues/499